CPPLINT.cfg

```diff
@@ -1,13 +1,14 @@
 set noparent
 headers=h,hpp,hxx
-root=sources
+extensions=cpp,h,hpp,hxx
+root=dynawo/sources
```

Comment:
dynawo/sources directory doesn't exist in DFL

Reply:
Fixed. The cpplint analysis still was performed because the cpplint wrapper is invoked with the proper source directory, and is responsible for building the list of files.